src/webui/api/torrentscontroller.cpp

```diff
@@ -268,6 +279,112 @@ namespace
 
         return url;
     }
+
+    QJsonObject serializePartialTorrentInfo(const BitTorrent::InfoHash &infoHash)
```

Comment:
Just name it by what it does directly: `serializeInfoHash`. PartialTorrentInfo term is unclear/ambigous.

Reply:
Will rename